web/workflow.R

```diff
@@ -39,29 +39,36 @@ if (is.na(args[1])){
   settings <- read.settings(settings.file)
 }
 
+# Update/fix/check settings. Will only run the first time it's called, unless force=TRUE
+settings <- fix.deprecated.settings(settings, force=FALSE)
+settings <- addSecrets(settings, force=FALSE)
+settings <- update.settings(settings, force=FALSE)
+settings <- check.settings(settings, force=FALSE)
+
 # Check for additional modules that will require adding settings
 if("benchmark" %in% names(settings)){
   library(PEcAn.benchmark)
   settings <- papply(settings, read.settings.RR)
 }
 
 # Write pecan.CHECKED.xml
-settings <- write.settings(settings, outputfile = "pecan.CHECKED.xml")
+write.settings(settings, outputfile = "pecan.CHECKED.xml")
```

Comment:
@rykelly We had discussed changing the name of `update.settings` to something different and then renaming `write.settings` as `update.settings` or I thought maybe `settings2pecan.xml`. I agree that  the name`write.settings` is misleading.